css/card.css

```diff
@@ -14,21 +14,30 @@ tcg-card, .card-component, .card-front,
   display: inline-block;
 }
 tcg-card {
-  margin: 3%;
+  margin: 1vmin;
   perspective: 600px; /* for the card animation */
 }
 
 #gacha-controls {
   --card-width: 400px; /* adjust the card size for the gacha page */
   --card-height: 559px; /* adjust the card size for the gacha page */
 }
-#gacha-controls #card-list {
+#gacha-controls #card-list,
+#collection-border #card-list {
+  position: relative;
   min-height: calc(var(--card-height) + 100px);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  row-gap: 2em;
+  column-gap: 1vmin;
 }
 #gacha-controls tcg-card {
   position: absolute;
   right:10%;
-  transition: all 0.3s ease; /* make any changes to this card flow, so it doesn't snap */
+  transition: transform 0.3s ease; /* make any changes to this card flow, so it doesn't snap */
```

Comment:
explicit definition to reduce browser load on possibly transitioning all css properties associated with this element